image/spec/specs-go/v1/image.go

```diff
@@ -0,0 +1,54 @@
+package v1
```

Comment:
LOL, I hate the `v1` package name, but it's consistent with the OCI-spec's structure, so 🤷‍♂️